src/Language/Haskell/HGrep.hs

```diff
@@ -25,14 +25,17 @@ module Language.Haskell.HGrep (
 
 
 import           Language.Haskell.HGrep.Internal.Data
+import           Language.Haskell.HGrep.Prelude
 import qualified Language.Haskell.HGrep.Print as HP
 import qualified Language.Haskell.HGrep.Query as HQ
-import           Language.Haskell.HGrep.Prelude
 
 import qualified Language.Haskell.GHC.ExactPrint as EP
 
 import qualified System.IO as IO
 
+import           GHC.Num ((+))
```

Comment:
This should have been included in the project Prelude, it's an oversight on my part. Sorry about that.